bazel_tools/client_server_test/tests/BUILD.bazel

```diff
@@ -0,0 +1,47 @@
+# Copyright (c) 2019 Digital Asset (Switzerland) GmbH and/or its affiliates. All rights reserved.
+# SPDX-License-Identifier: Apache-2.0
+
+load(
+  "//bazel_tools/client_server_test:client_server_test.bzl",
+  "client_server_test"
+)
+
+load(
+  "//bazel_tools:scala.bzl",
+  "da_scala_binary",
+)
+
+sh_binary(
+  name = "client",
+  srcs = ["client.sh"],
+)
+
+exports_files(["client_input_file"])
+
+da_scala_binary(
+  name = "server",
+  main_class = "Main",
+  srcs = ["server.scala"],
+  deps = [
+    "//3rdparty/jvm/com/github/scopt:scopt",
+  ],
+)
+
+client_server_test(
+  name = "test",
+  client = ":client",
+  client_args = [
+    "--foobar",
+    # Use the input file passed in via `data`.
+    #"bazel_tools/client_server_test/tests/client_input_file"
+    "$(location :client_input_file)"
+  ],
+  server = ":server",
```

Comment:
Will server args be supported?

Reply:
They are: https://github.com/digital-asset/daml/pull/365/files#diff-6c30fe5cafc51117473d933c4db9e94aR24